index.js

```diff
@@ -45,17 +45,33 @@ module.exports = opts => {
 	const headers = opts.verbose ? verboseHeaders : defaultHeaders;
 
 	return pify(childProcess.execFile)('tasklist', args)
-		.then(stdout => neatCsv(stdout, {headers}))
-		.then(data => data.map(task => {
-			// Normalize task props
-			task.pid = Number(task.pid);
-			task.sessionNumber = Number(task.sessionNumber);
-			task.memUsage = Number(task.memUsage.replace(/[^\d]/g, '')) * 1024;
-
-			if (opts.verbose) {
-				task.cpuTime = sec(task.cpuTime);
-			}
-
-			return task;
-		}));
+		// INFO: No tasks are running which match the specified criteria.
+		// See: https://github.com/sindresorhus/tasklist/issues/9
+		.then(stdout => stdout.startsWith('INFO:') ?
+			[] :
+			neatCsv(stdout, {headers})
```

Comment:
Including indentation, that line would be longer than 80 characters, which is too long IMHO. That's why I added the line breaks.